spec/app/ics-031-crosschain-queries/README.md

```diff
@@ -163,17 +163,17 @@ function queryPath(id: Identifier): Path {
 The result query path is a private path that stores the result of completed queries.
 
 ```typescript
-function resultQueryPath(id: Identifier): Path {
-    return "queriesresult/{id}"
+function queryResultPath(id: Identifier): Path {
+    return "queries/{id}/result"
```

Comment:
Isn't weird that the path ends with `result` instead of a function argument {}?

Reply:
To me it doesn't look strange, and we also follow a similar pattern in [other paths](https://github.com/cosmos/ibc/blob/main/spec/core/ics-024-host-requirements/README.md#path-space).